lua/bufferline/diagnostics.lua

```diff
@@ -73,7 +73,18 @@ end
 local get_diagnostics = {
   nvim_lsp = function()
     if is_nightly then
-      return vim.diagnostic.get()
+      results = {}
```

Comment:
This should be
```suggestion
      local results = {}
````
Otherwise the `results` variable will be a global variable and almost certainly clash with something

Reply:
Fixed now, sorry about that.. I'm used to writing moonscript :P